### PR TITLE
ADCM-2741 add checks for group configs

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

**Comment by annademina** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @annademina! Build has been launched!

File: tests/ui_tests/test_cluster_list_page.py [hunk 1]
```diff
@@ -1393,20 +1393,21 @@ def test_check_pagination_on_group_config_component_page(self, app_fs, create_co
...
+        ids=("config_group_customization_true", "config_group_customization_false", "no_config_group_customization"),
     )
     @pytest.mark.parametrize(
-        "group_customization", [True, False], ids=("group_customization_true", "group_customization_false")
+        "group_customization",
+        [True, False, None],
+        ids=("group_customization_true", "group_customization_false", "no_group_customization"),
     )
     @pytest.mark.usefixtures("login_to_adcm_over_api")
     def test_group_config_fields_invisible_false(
```
Comment (Sealwing) [inline tests/ui_tests/test_cluster_list_page.py:1413]: Don't forget to return it to full

File: tests/ui_tests/test_cluster_list_page.py [hunk 2]
```diff
@@ -1464,7 +1480,124 @@ def check_expectations():
...
+        [True, False, None],
+        ids=("group_customization_true", "group_customization_false", "no_group_customization"),
+    )
+    @pytest.mark.parametrize(
+        "field_customization",
+        [True, False, None],
+        ids=("field_customization_true", "field_customization_false", "no_field_customization"),
+    )
+    @pytest.mark.usefixtures("login_to_adcm_over_api")  # pylint: disable-next=too-many-locals
+    def test_group_configs_fields_in_group_invisible_false(
```
Comment (Sealwing) [inline tests/ui_tests/test_cluster_list_page.py:1510]: This one should be full too, I think

**Comment by BenderArenadata** [comment]
Hey, @annademina! Build has been launched!

File: tests/ui_tests/test_cluster_list_page.py (hunk 1 above)
Comment (annademina) [reply tests/ui_tests/test_cluster_list_page.py]: returned

File: tests/ui_tests/test_cluster_list_page.py (hunk 2 above)
Comment (annademina) [reply tests/ui_tests/test_cluster_list_page.py:1539]: returned

**Review by Sealwing — APPROVED** [review]

**Pull request merged**